docs/pages/cairo/hello-dojo.md

```diff
@@ -93,20 +96,20 @@ mod actions {
     use starknet::{ContractAddress, get_caller_address};
     use dojo_starter::models::{position::{Position, Vec2}, moves::{Moves, Direction}};
 
-    // declaring custom event struct
-    #[event]
-    #[derive(Drop, starknet::Event)]
-    enum Event {
-        Moved: Moved,
-    }
-
-    // declaring custom event struct
-    #[derive(starknet::Event, Model, Copy, Drop, Serde)]
-    struct Moved {
-        #[key]
-        player: ContractAddress,
-        direction: Direction
-    }
+    // declaring custom starknet event
+    // #[event]
+    // #[derive(Drop, starknet::Event)]
+    // enum Event {
+    //     Moved: Moved,
+    // }
+
+    // declaring custom starknet event struct
+    // #[derive(starknet::Event, Model, Copy, Drop, Serde)]
+    // struct Moved {
+    //     #[key]
+    //     player: ContractAddress,
+    //     direction: Direction
+    // }
```

Comment:
Why is this commented only? We can remove it from here, or did you have something else in mind?